chapters/en/slides/module4/module4_04.md

```diff
@@ -0,0 +1,131 @@
+---
+type: slides
+title: 'module4_04'
+---
+
+
+# Handling merge conflicts
+
+
+
+---
+# What is a merge conflict?
+
+Error message that indicates that there are changes on the remote repository that you do not have locally. 
```

Comment:
But is this a "merge conflict" per se? It is an indication that the local repo is not up to date, but there may not necessarily be a conflict.